Net/SMTP.php

```diff
@@ -1,9 +1,27 @@
 <?php
-/** vim: set expandtab softtabstop=4 tabstop=4 shiftwidth=4: */
+/**
+ * Provides an implementation of the SMTP protocol using PEAR's
+ * Net_Socket class.
+ *
+ * PHP version 5
+ *
+ * @category Network
+ * @package  Net_SMTP
+ * @author   Chuck Hagenbuch <chuck@horde.org>
+ * @author   Jon Parise <jon@php.net>
+ * @author   Damian Alejandro Fernandez Sosa <damlists@cnba.uba.ar>
+ * @author   Armin Graefe <schengawegga@gmail.com>
+ * @license  http://opensource.org/licenses/bsd-license.php BSD-2-Clause
+ * @link     https://pear.php.net/package/Net_SMTP/
+ *
+ * @example basic.php A basic implementation of the Net_SMTP package.
+ *
+ * vim: set expandtab softtabstop=4 tabstop=4 shiftwidth=4: 
+ */
 // +----------------------------------------------------------------------+
```

Comment:
The license block is traditionally the first thing in the source file.

Reply:
Yes, but PHP_CodeSniffer will raise an error if the ClassDoc is not the first thing in the source file. I am not very happy with some things in PHP_CodeSniffer, too. For example, PHP_CodeSniffer requests a doc area at the first position in the file AND a doc area directly above the class opening. Thats why the class doc area is doubled in the file. PHP_CodeSniffer also does not accept the "&" in the doc area for reference variable passing thru als parameter. I can change these things in the file, but then we will raise errors on PHP_CodeSniffer. I think, this will be acceptable. What do you think about?

Reply:
I think it's fine to take the changes you like and ignore the more disruptive ones.

Is the tool configurable?

Reply:
Yes, it is configurable in that you can set the default coding standard, like PEAR or PSR-1, for example. If no standard is manually set, PEAR will be predefined as standard. I will check, if the checks passes when i integrate the license in the class doc area.

Reply:
It worked. It seems like it doesn´t matter what is written at first in the first class doc area. a few "@" statements are necessary in this doc area, but it doesn´t matter at wich position they are written.

I pushed the changed commit again.

Reply:
I think the result still isn't that great because it adds the license text to the `Net_SMTP` package's documentation block, and that will cause it to show up in documentation tools, etc.